ocp_resources/storage_profile.py

```diff
@@ -4,4 +4,33 @@
 
 
 class StorageProfile(Resource):
+    """
+    StorageProfile Object
+
+    Doc:
+    https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md
+    """
+
     api_group = Resource.ApiGroup.CDI_KUBEVIRT_IO
+
+    @property
+    def claim_property_sets(self):
+        return self.instance.status.get("claimPropertySets")
+
+    def first_claim_property_set_access_modes(self):
+        return self.claim_property_sets[0].get("accessModes") if self.claim_property_sets else None
+
+    def first_claim_property_set_volume_mode(self):
+        return self.claim_property_sets[0].get("volumeMode") if self.claim_property_sets else None
```

Comment:
I see that `self.claim_property_sets` is list, you should return the list and not list[0]

Reply:
Its a list of objects
Each object has accessmodes & volumemode
And the property `claim_property_sets` returns a list already

The idea of this function is to return the accessmode/volumemode of the very first object in the list, usually for a  built-in StorageProfile that we expected for atleast one object to exist

In this scenario, if `self.claim_property_sets[0]` fails - then something is went wrong

Reply:
ok, saw you rename the function.